sumologic/resource_sumologic_generic_polling_source.go

```diff
@@ -125,6 +125,11 @@ func resourceSumologicGenericPollingSource() *schema.Resource {
 					Type:     schema.TypeString,
 					Optional: true,
 				},
+				"use_versioned_api": {
+					Type:     schema.TypeBool,
+					Optional: true,
+					Default:  false,
```

Comment:
In UI we have true as default value though, which one is correct?

Reply:
the default is `true`. Fixed.